### ISSUE TITLE
Rule Request: Strong IBOutlet

### ISSUE THREAD

**Issue by sindresorhus** [realm/SwiftLint#2433]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### New rule request

1. Why should this rule be added? Share links to existing discussion about what
   the community thinks about this.

Enforce not using `weak` for `@IBOutlet`. Even though Xcode defaults to `weak`, [Apple has recommended using strong outlets since WWDC 2015](https://stackoverflow.com/a/31395938/64949).

2. Provide several examples of what _would_ and _wouldn't_ trigger violations.

Would trigger:

```swift
final class ViewController: NSViewController {
	@IBOutlet weak var label: NSTextField!
}
```

Would not trigger:

```swift
final class ViewController: NSViewController {
	@IBOutlet var label: NSTextField!
}
```

```swift
final class ViewController: NSViewController {
	weak var label: NSTextField = …
}
```

3. Should the rule be configurable, if so what parameters should be configurable?

No

4. Should the rule be opt-in or enabled by default? Why?
   See [README.md](../README.md#opt-in-rules) for guidelines on when to mark a rule as opt-in.

I think it should be enabled by default as it adheres to the guidelines and it's what Apple recommends. In the few cases you would actually want it to be weak you can just use a `// swiftlint:disable` comment.
